vllm/engine/llm_engine.py

```diff
@@ -662,7 +662,7 @@ def _get_stats(
             if scheduler_outputs.num_prefill_groups > 0 and len(
                     scheduler_outputs.scheduled_seq_groups
             ) != scheduler_outputs.num_prefill_groups:
-                print("DETECTED CHUNKED")
+                logger.info("DETECTED CHUNKED")
```

Comment:
@rkooo567 can you suggest a better message here, and should it be debug?

Reply:
I feel like we don't need this at all? If we want to keep this, I think just making it debug without changing msg seems okay to me

Reply:
ok can you send a pr to remove?

Reply:
@simon-mo The debug message has been removed and left with a comment.